### ISSUE TITLE
offline cache

### ISSUE THREAD

**Issue by eadmaster** [anthonycr/Lightning-Browser#120]
is it possible to view recently-visited pages even if the internet connection drops?

**Comment by Fuseteam** [comment]
is it possible to save the cache to sd card if available?

**Comment by easyaspi314** [comment]
I was thinking that maybe we could use [`WebView.saveWebArchive(String filename)`](http://developer.android.com/reference/android/webkit/WebView.html#saveWebArchive%28java.lang.String%29) and access the way [gregko explains it.](http://stackoverflow.com/a/13445760)

**Comment by anthonycr** [comment]
@easyaspi314 yes that is indeed a way to save the page. I think that could be a solution if I added a "save page" option like you mentioned.

Maybe I can add in some code that checks the network connection and if it's not there then I load using CACHE_ELSE_NETWORK option. I used to use it in Lightning, but switched for some reason I can't remember (likely was a bad reason).

**Comment by klawdhfzasjhaa** [comment]
+1 for this feature
